packages/core/src/payment-integration/default-payment-integration-service.ts

```diff
@@ -96,4 +100,20 @@ export default class DefaultPaymentIntegrationService implements PaymentIntegrat
 
         return this._storeProjection.getState();
     }
+
+    async signOut(methodId: string, options?: RequestOptions): Promise<PaymentIntegrationSelectors> {
```

Comment:
I would prefer not to expose any `/remote-checkout/*` as it is private and maintained by integration team. These endpoints can be called directly as they don't result in states that need to be managed via `CheckoutStore`.

Reply:
So shall we remove this from the service and inject it directly when the strategy is constructed.